cpp/include/cuml/neighbors/knn.hpp

```diff
@@ -94,12 +94,12 @@ void knn_classify(cumlHandle &handle, int *out, int64_t *knn_indices,
  * @param out output array on device (size n_samples)
  * @param knn_indices array on device of knn indices (size n_samples * k)
  * @param y array of labels on device (size n_samples)
- * @param n_labels number of vertices in index (eg. size of each y array)
+ * @param n_index_rows number of vertices in index (eg. size of each y array)
```

Comment:
should this be n_query_rows?

Reply:
No, each `y` array contains the labels of size `n_index_rows` that correspond to the column values in `knn_indices`. `n_query_rows` is the number of rows in `knn_indices`.